uint8 priority # 0: Low priority; 255: High priority
EndCondition end_condition
